adagrams/game.py

```diff
@@ -1,11 +1,69 @@
+import random
+
 def draw_letters():
-    pass
+    #letter pool is a dictionary with letters as keys and quanity values
+    letter_pool = {'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3, 'H': 2, 'I': 9, 'J': 1, 'K': 1, 
+                   'L': 4, 'M': 2, 'N': 6, 'O': 8, 'P': 2, 'Q': 1, 'R': 6, 'S': 4, 'T': 6, 'U': 4, 'V': 2, 
+                   'W': 2, 'X': 1, 'Y': 2, 'Z': 1}
+
+    hold_letters = []
+
+    while len(hold_letters) < 10:
+        letter = random.choice(list(letter_pool.keys()))
+
+        if letter_pool[letter] > 0:
+            hold_letters.append(letter)
+            letter_pool[letter] -= 1
```

Comment:
So we want to make sure that we are making a list of letters we the appropriate weights (if we have `"A": 4` then that should correspond to `["A', "A", "A", "A"]`. We could do some with an implementation could look something like this:
```python
letters = []
for letter in LETTER_POOL: # this will give us the keys
   letters += letter * LETTER_POOL[letter]
```

then we could get our 10 like so:

```python 
 hold_letters = random.sample(letters, 10)   
```